cdc/sink/mq_flush_worker.go

```diff
@@ -43,21 +43,23 @@ type mqEvent struct {
 
 // flushWorker is responsible for sending messages to the Kafka producer on a batch basis.
 type flushWorker struct {
-	msgChan    chan mqEvent
+	msgChan       chan mqEvent
+	ticker        *time.Ticker
```

Comment:
When is it closed?

Reply:
I assume it shuts down when an error occurs or the flushworker is stopped? Because when an error occurs during the run, it is reported to the processor, which then tries to shut down the sink.